packages/ses/src/compartment-shim.js

```diff
@@ -120,6 +202,58 @@ export const CompartmentPrototype = {
   __isKnownScopeProxy__(value) {
     return privateFields.get(this).knownScopeProxies.has(value);
   },
+
+  module(specifier) {
+    if (typeof specifier !== 'string') {
+      throw new TypeError('first argument of module() must be a string');
+    }
+
+    assertModuleHooks(this);
+
+    const { exportsProxy } = getDeferredExports(
+      this,
+      privateFields.get(this),
+      moduleAliases,
+      specifier,
+    );
+
+    return exportsProxy;
+  },
+
+  async import(specifier) {
+    if (typeof specifier !== 'string') {
+      throw new TypeError('first argument of import() must be a string');
+    }
+
+    assertModuleHooks(this);
+
+    return load(privateFields, moduleAliases, this, specifier).then(() => {
+      const namespace = this.importNow(specifier);
+      return { namespace };
```

Comment:
Are the curlies here because of the Node thenable avoidance issue that we'd like to renegotiate? Do we still wish to renegotiate it? If so, please add a comment warning the reader of this possible incompat change.

Reply:
Yes. This is likely to change.